api/db/dbHelper.js

```diff
@@ -19,22 +20,25 @@ function getSequelize() {
 
 sequelize = getSequelize();
 
-var Products = sequelize.define('product', ProductsDataModel, {freezeTableName: true});
-var Prices = sequelize.define('price', PricesDataModel, {freezeTableName: true})
-var Customers = sequelize.define('customer', CustomersDataModel, {freezeTableName: true});
-var Orders = sequelize.define('order', OrdersDataModel, {freezeTableName: true});
+var Products = sequelize.define('product', ProductsDataModel, { freezeTableName: true });
+var Prices = sequelize.define('price', PricesDataModel, { freezeTableName: true })
+var Customers = sequelize.define('customer', CustomersDataModel, { freezeTableName: true });
+var Orders = sequelize.define('order', OrdersDataModel, { freezeTableName: true });
+
+var ProductsOrders = sequelize.define('products_orders', { quantity: Sequelize.INTEGER })
 
 Products.Price = Products.belongsTo(Prices, { onDelete: 'cascade' });
-Orders.Customers = Orders.belongsTo(Customers, { onDelete: 'cascade' });
-Customers.Orders = Customers.hasMany(Orders, { as: 'Orders' });
+Orders.Customer = Orders.belongsTo(Customers, { onDelete: 'cascade' });
+Customers.Order = Customers.hasMany(Orders, { as: 'orders' });
 
-Products.belongsToMany(Orders, {through: 'products_orders', foreignKey: 'productId'});
-Orders.belongsToMany(Products, {through: 'products_orders', foreignKey: 'orderId'});
+Orders.belongsToMany(Products, { as: 'products', through: ProductsOrders, foreignKey: 'orderId' })
+Products.belongsToMany(Orders, { as: 'orders', through: ProductsOrders, foreignKey: 'productId' })
 
 sequelize.sync({ force: true });
 
 exports.Prices = Prices;
 exports.Products = Products;
 exports.Customers = Customers;
 exports.Orders = Orders;
-exports.Sequelize = sequelize;
\ No newline at end of file
+exports.Sequelize = sequelize;
```

Comment:
can u keep exports.Sequelize at the very bottom or at the very top of the whole exports part? So that it's easy to find it anytime